doc/piTest.1

```diff
@@ -1,6 +1,4 @@
 .\"                                      Hey, EMACS: -*- nroff -*-
-.\" (C) Copyright 2016-2018 Kunbus GmbH
```

Comment:
Why don't we use an SPDX header her?

Reply:
Because we can't: The reuse tool doesn't recognize comments for roff files, so a `.license` file would need to be used. Instead of possibly littering the repository with those in several places in the future this is done in the `.reuse/dep5` file instead. So you can find the license information for that file in `.reuse/dep5`, which is compliant with REUSE.